src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable react/display-name */
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useState } from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoField } from '../TodoField';
+
+type Props = {
+  todos: Todo[];
+  deleteTodo: (id: number) => void;
+  changeCompleted: (id: number) => void;
+  changeTodo: (id: number, newTitle: string) => void;
+};
+
+export const TodoList: React.FC<Props> = ({
+  todos,
+  deleteTodo,
+  changeCompleted,
+  changeTodo,
+}) => {
+  const [selectedTodoId, setSelectedTodoId] = useState<number | null>(null);
```

Comment:
probably better to have all states in app so that there is one smart component and the rest are engaged in rendering